src/components/MainView/main-view.jsx

```diff
@@ -49,16 +58,117 @@ console.log(movies);
   }
```

Comment:
- The lines of code from 45 to 58 are unnecessary. Determining whether a view is rendered or not is handled by the react-router.